### PR TITLE
Add UI feature with working example and update documentation

### PR DESCRIPTION
**Pull request by p-patel**
### What

Add UI feature with working example and update documentation.

### How to review

Review code, example and updated instructions to test UI features.

### Who can review

Anyone

### CODE REVIEW

File: examples/ui/main.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+func ExampleHandler1(w http.ResponseWriter, r *http.Request) {
+	htmlPage := "<!doctype html><html lang=en><head><meta charset=utf-8><title>blah</title></head><body><p class='example-paragraph'>Example web page</p></body></html>"
+	fmt.Fprint(w, htmlPage)
```
Comment (dandwal) [inline examples/ui/main.go:13]: Fprint returns two arguments right? The second argument is a possible error. Aware this is an example so not a must change comment, and can also see in other example tests (from before this PR) there are errors missed too. This is just me 2 cents.

File: examples/ui/myapp_component.go [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+		ServiceRunning: false,
+	}
+
+	c.run(ctx)
+	c.ServiceRunning = true
+
+	return c
+}
+
+func (c *MyAppComponent) run(ctx context.Context) {
```
Comment (dandwal) [inline examples/ui/myapp_component.go:53]: Is contexct supposed to be used, or is the argument not needed?

**Review by dandwal — COMMENTED** [review]
Not sure what the overall criteria for this task was, should we have examples on interaction? Think right now we just have an example of on page load x and y should be visible. - which might be enough, what are your thoughts?

File: README.md [hunk 3]
```diff
@@ -85,11 +89,70 @@ func TestComponent(t *testing.T) {
...
+package main
+
+import (
+	componenttest "github.com/ONSdigital/dp-component-test"
+)
+
+var componentFlag = flag.Bool("component", false, "perform component tests")
+
+func InitializeScenario(ctx *godog.ScenarioContext) {
+	myAppComponent := NewMyAppComponent() // This is the part that YOU will implement
```
Comment (joegasewicz) [inline README.md:108]: Is this call missing a  `handler http.Handler`arg?

File: examples/ui/myapp_component.go (hunk 2 above)
Comment (p-patel) [reply examples/ui/myapp_component.go]: Removed

File: ci/scripts/component.sh [hunk 4]
```diff
@@ -1,7 +1,10 @@
 #!/bin/bash -eux
 
-cwd=$(pwd)
+wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
+echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list
```
[the remark this answers is not in the document]
Comment (p-patel) [reply ci/scripts/component.sh:4]: Sounds like a good idea - tempted to do that as a separate ticket. Can run it passed the rest of the team as an optimisation and will avoid holding up this ticket and potentially blocking the ticket that's adding the UI component tests to Area Profiles. What do you think?

[the remark this answers is not in the document]
Comment (dandwal) [reply ci/scripts/component.sh:4]: Sure yer something we can talk about in standup maybe

**Review by dandwal — APPROVED** [review]
Approved, we just need to checkin around the docker image used

**Comment by p-patel** [comment]
> Approved, we just need to checkin around the docker image used

Confirmed, image will be updated in a separate ticket. Ticket has been created.

**Review by dandwal — APPROVED** [review]

**Pull request merged**